formats/json/api/kotlinx-serialization-json.api

```diff
@@ -358,3 +358,43 @@ public final class kotlinx/serialization/json/JvmStreamsKt {
 	public static final fun encodeToStream (Lkotlinx/serialization/json/Json;Lkotlinx/serialization/SerializationStrategy;Ljava/lang/Object;Ljava/io/OutputStream;)V
 }
 
+public final class kotlinx/serialization/json/internal/JsonLexerKt {
+	public static final field BATCH_SIZE I
+}
+
+public final class kotlinx/serialization/json/internal/JsonStreamsKt {
+	public static final fun decodeByReader (Lkotlinx/serialization/json/Json;Lkotlinx/serialization/DeserializationStrategy;Lkotlinx/serialization/json/internal/SerialReader;)Ljava/lang/Object;
+	public static final fun decodeToSequenceByReader (Lkotlinx/serialization/json/Json;Lkotlinx/serialization/json/internal/SerialReader;Lkotlinx/serialization/DeserializationStrategy;Lkotlinx/serialization/json/DecodeSequenceMode;)Lkotlin/sequences/Sequence;
+	public static synthetic fun decodeToSequenceByReader$default (Lkotlinx/serialization/json/Json;Lkotlinx/serialization/json/internal/SerialReader;Lkotlinx/serialization/DeserializationStrategy;Lkotlinx/serialization/json/DecodeSequenceMode;ILjava/lang/Object;)Lkotlin/sequences/Sequence;
+	public static final fun encodeByWriter (Lkotlinx/serialization/json/Json;Lkotlinx/serialization/json/internal/JsonWriter;Lkotlinx/serialization/SerializationStrategy;Ljava/lang/Object;)V
+}
+
+public abstract interface class kotlinx/serialization/json/internal/JsonWriter {
+	public abstract fun release ()V
+	public abstract fun write (Ljava/lang/String;)V
+	public abstract fun writeChar (C)V
+	public abstract fun writeLong (J)V
+	public abstract fun writeQuoted (Ljava/lang/String;)V
+}
+
+public abstract interface class kotlinx/serialization/json/internal/SerialReader {
+	public abstract fun read ([CII)I
+}
+
+public final class kotlinx/serialization/json/internal/StreamingJsonDecoderKt {
+	public static final fun decodeStringToJsonTree (Lkotlinx/serialization/json/Json;Lkotlinx/serialization/DeserializationStrategy;Ljava/lang/String;)Lkotlinx/serialization/json/JsonElement;
+}
+
+public final class kotlinx/serialization/json/internal/StringOpsKt {
+	public static final fun getESCAPE_MARKERS ()[B
```

Comment:
I think this definitely should be converted to `@PublishedApi internal`. While other internal API can be potentially implemented by very curious users willing to implement their own stream adaptation (and we're ok with that) and can't mess our implementation much, these vals are giving access to internal data structures (and given that `Array` is mutable, may do very bad things)

I've tried to use `@file:Suppress` in various places and it worked normally, so I can push this change myself after you finish with other fixes